cmd/crictl/attach.go

```diff
@@ -63,9 +68,10 @@ var runtimeAttachCommand = &cli.Command{
 		defer cancel()
 
 		var opts = attachOptions{
-			id:    id,
-			tty:   c.Bool("tty"),
-			stdin: c.Bool("stdin"),
+			id:        id,
+			tty:       c.Bool("tty"),
+			stdin:     c.Bool("stdin"),
+			websocket: c.Bool("websocket"),
```

Comment:
would the name `use-websocket` be better? Or maybe have something with three options: `-- transport=spdy|websocket|default` where default will do websocket with the fallback to spdy?

Reply:
fixed